integration/components/UserInfo.tsx

```diff
@@ -155,6 +187,34 @@ export default class UserInfo extends React.Component<Record<string, never>, Use
         );
     }
 
+    getDeviceListContent() {
+        if (this.state.loading) {
+            return <LoadingPlaceholder />;
+        }
+
+        if (this.state.deviceList.length !== 0) {
+            return (
+                <div>
+                    {this.state.deviceList
+                        .map((device, index) => (
+                            <ul key={index}>
+                                <li>Name: {device.name}</li>
+                                <li>ID: {device.id}</li>
+                                <li>Current Device: {device.isCurrentDevice}</li>
+                                <li>Public Signing Key: {device.publicSigningKey}</li>
+                                <li>Created: {device.created}</li>
+                                <li>Updated: {device.updated}</li>
+                            </ul>
+                        ))
+                        .flatMap((e, index) => [<hr key={index}/>, e])
+                        .slice(1)}
```

Comment:
What does this slice do?

Reply:
Drops the first element off. `shift` is the other way to do it, but it returns the dropped element, not the array.

Reply:
```
> [1,2,3,4].flatMap((e,index) => ['+', e])
(8) ['+', 1, '+', 2, '+', 3, '+', 4]
> [1,2,3,4].flatMap((e,index) => ['+', e]).slice(1)
(7) [1, '+', 2, '+', 3, '+', 4]
```